submissions/TaniaBondarenko/a-tiny-js-world-update/index.js

```diff
@@ -0,0 +1,75 @@
+class Inhabitant {
+  constructor(name, gender, saying, friends,legs, hands) {
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends;
+    this.legs = legs;
+    this.hands = hands;
```

Comment:
Now `Animal` class inherits hands from `Inhabitant` class, thus all animals have hands.
Hands are unique to humans. So it is `Human` that has to introduce `hands` property and handle it solely.

Reply:
I think, I got it. Thank you.